plugins/logs/plugin.go

```diff
@@ -91,9 +93,7 @@ func (c *Config) validateAndInjectDefaults(services []string, plugins []string)
 		if !found {
 			return fmt.Errorf("invalid plugin name %q in decision_logs", *c.Plugin)
 		}
-	} else if c.Service == "" && len(services) != 0 {
```

Comment:
Right, I brought this up previously 😄 

The problem is that if you had a service that isn't a decision logger and you want to enable the console logging of decisions it would just try and use the first service defined for decisions too.

We could say that behavior is mutually exclusive with the console option being enabled. Something like if `decsion_logs.console` is true then you _must_ specify a service name, but otherwise we default to the first service?

It would keep existing configurations working but makes for, IMO, confusing behavior. Even the existing behavior is kinda magic since it isn't documented anywhere.

Reply:
I agree the current behaviour should be better documented.

IMO, console logging can be mutually exclusive with the others (like custom loggers are mutually exclusive with the batched/HTTP logger.)